Adding Stress, a new item, minor update to inventory resource.lua, and added yoga

iTzCrutchie opened the pull request:
- Renamed fsn_hungerandthirst to fsn_needs
- Added Stress to fsn_needs
- Added gaining stress to bank heists, streetselling and robbing houses.
- I might have missed other areas to add stress but just need to add to the client side the following to add or remove stress
- TriggerEvent('fsn_stress:stressingOut', 5)
- TriggerEvent('fsn_stress:removeStress', 5)
- The number can be changed to however much stress you want to add or remove.
- Added cigarettes to items as well as its corresponding use and image
- changed how the resource.lua in inventory looks for items. instead ofl isting each indivual png it now just uses a wildcard *.png
- Added a new drug effect for the cigarette on use
- Changed smoking a joint effect to also reduce stress.
- Added Yoga activity to reduce stress.

-- If anything else got broken, i dont think it did, but if anything did i will update it. So far it has been working for me with no errors

Commit:
Merge pull request #1 from jamessc0tt/master

Bring up to latest commits

Commit:
Merge pull request #2 from jamessc0tt/master

Menu Update (#5)

Commit:
fix __resource.lua

Fix __resource.lua on fsn_bank and fsn_cargarage after the update for the settings file in main.

Commit:
Adding Stress, a new item, minor update to inventory resource.lua, and added yoga activity

- Renamed fsn_hungerandthirst to fsn_needs
- Added Stress to fsn_needs
-Added gaining stress to bank heists, streetselling and robbing houses.
-I might have missed other areas to add stress but just need to add to the client side the following to add or remove stress
- TriggerEvent('fsn_stress:stressingOut', 5)
-TriggerEvent('fsn_stress:removeStress', 5)
- The number can be changed to however much stress you want to add or remove.
- Added cigarettes to items as well as its corresponding use and image
- changed how the resource.lua in inventory looks for items. instead ofl isting each indivual png it now just uses a wildcard *.png
- Added a new drug effect for the cigarette on use
- Changed smoking a joint effect to also reduce stress.
- Added Yoga activity to reduce stress.

-- If anything else got broken, i dont think it did, but if anything did i will update it. So far it has been working for me with no errors

jamessc0tt commented:
Please remove replicated KEYS table:
https://github.com/jamessc0tt/FiveM-FSN-Framework/pull/12/commits/d1d828b4e6cad00b59cf24ce73d7c83a5a15ebd3#diff-a419d5741f74c62051b5183f0c9a8756R1

jamessc0tt commented:
Please rename 'fsn_stress:stressingOut' to 'fsn_stress:stress:add' and 'fsn_stress:stress:remove' or something uniform with the rest of the server.

Commit:
Update stress event name, remove keys from yoga client.lua

- Update stress even names to fsn_needs:stress:add and fsn_needs:stress:remove
- Removed local Keys from yoga client.lua
- Updated stress events in other scripts to reflect the new event name for adding and removing stress

Commit:
update variable name on stress add to increase

- Update stress variable for adding stress from anxiet to increase

iTzCrutchie commented:
Done. also updated the variable name to better reflect that it is adding stress instead of just calling it anxiety. If there are any more changes that you need me to make let me know. Thanks!
